hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileSystem.java

```diff
@@ -1623,35 +1615,6 @@ public void testGetTrashRoots() throws IOException {
     Assert.assertEquals(6, res.size());
   }
 
-  /**
-   * Check that files are moved to trash.
-   * since fs.rename(src,dst,options) is enabled.
-   */
-  @Test
-  @Flaky("HDDS-6646")
-  public void testRenameToTrashEnabled() throws Exception {
```

Comment:
Was the test removal intended? because earlier commit had fixed the test and description also says so.

Reply:
> Was the test removal intended? because earlier commit had fixed the test and description also says so.

Yes @sadanand48 , as [HDDS-6645](https://issues.apache.org/jira/browse/HDDS-6645) will be taking care of testTrash test case fix, so the only assertion which was different from `testRenameToTrashEnabled` and `testTrash` has been moved to  `testTrash`  and `testRenameToTrashEnabled` has been removed as per suggestion also from @sumitagrawl